### DISCUSSION TITLE
Structured logging scope appearing as @Properties

### DISCUSSION THREAD

**Discussion by TheMagnificent11** [datalust/seq-tickets#2024] [Help] [answered]
We are using Seq for local development, but not in production.

I've noticed some strange behaviour where a logging scope `common-request-id` is added using the `@Properties` syntax, while `cid` has been created the same way and appears as a normal structured logging property.

I've got following code that stores some logging scopes on our "correlation context accessor".

```
public static void SetCorrelationIds(this ICorrelationContextAccessor correlationContextAccessor, string commonIdServicePrefix)
{
    var cid = CorrelationIdFactory.GenerateCid();
    var commonRequestId= $"{commonIdServicePrefix}|{cid}";

    correlationContextAccessor.CorrelationContext = new CorrelationContext
    {
        LoggingProperties = new Dictionary<string, object>
        {
            { "cid", cid },
            { "common-request-id", commonRequestId }
        }
    };
}
```

I then have a gRPC interceptor that adds this dictionary as logging scopes.

```
public override async Task<TResponse> UnaryServerHandler<TRequest, TResponse>(
   TRequest request,
    ServerCallContext context,
    UnaryServerMethod<TRequest, TResponse> continuation)
{
    ....

    using (logger.BeginScope(correlationContext.LoggingProperties))
    {
        return await continuation(request, context);
    }
}
```

But, in Seq, `cid` appears how I expect it, but `common-request-id` appears as a `@Properties`, even though they are logged the same way.

Does it have something to do with the pipe in the `common-request-id` value?

![image](https://github.com/datalust/seq-tickets/assets/5241102/b8d11c2c-1acf-4ee3-88ad-054058dc6308)

**Comment by nblumhardt** [comment, marked as answer]
Hi @TheMagnificent11, thanks for your message.

The inlay hint is showing you how the property needs to appear when used in queries, since the name is irregular (includes dashes, which are the subtraction operator in query syntax).

You can hover over the inlay hint to see this.

Hope this helps!
Nick

**Reply by TheMagnificent11** [reply]
Thanks @nblumhardt
